test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewEngineTest.cs

```diff
@@ -594,7 +594,109 @@ public void FindPage_ReturnsSearchedLocationsIfPageCannotBeFound()
             Assert.Equal(expected, result.SearchedLocations);
         }
 
-       [Fact]
+        [Theory]
+        // Looks in RouteValueDefaults
+        [InlineData(true)]
+        // Looks in RouteConstraints
+        [InlineData(false)]
+        public void FindPage_SelectsActionCaseInsensitively(bool isAttributeRouted)
+        {
+            // The RouteData contains "Foo", and the ActionDescriptor contains "foo"
```

Comment:
Reverse these imo, that would make it much more like the actual scenario users reported.